ui/app/AppLayouts/Profile/Sections/Contacts/Contact.qml

```diff
@@ -46,4 +47,11 @@ Rectangle {
         font.pixelSize: 15
         color: Theme.darkGrey
     }
+    RadioButton {
+        visible: selectable
```

Comment:
Didn't you need the `id` for this to work?

Reply:
nop